index.js

```diff
@@ -11,11 +11,13 @@ module.exports = {
       ignoreComments: true,
       ignoreStrings: true,
     }],
+    'import/extensions': 'off',
```

Comment:
Can this be bigmaven specific? I actually like to be explicit about my extensions because all other types are explicit (`.css`, `.coffee`, ...) If you don't like it here, I can turn it on specifically for `mavenlink-ui`